frontend/app/controllers/Info.scala

```diff
@@ -66,7 +67,17 @@ trait Info extends Controller {
     )
   }
 
-  def supporterAustralia = CachedAndOutageProtected { implicit request =>
+  def supporterAustralia = NoCacheAction { implicit request =>
```

Comment:
Thanks for flagging this @svillafe. I'm going to discuss with @davidfurey tomorrow when he has some time and come to decision as to whether we need to cache or not. The other thing to consider is that we're planning to do the same with US supporter page, so we'll need to look at the traffic to that page too - it might mean that we can only run one of the A/B tests at a time